nemo/collections/nlp/models/text_classification/ptune_text_classification_model.py

```diff
@@ -29,7 +29,7 @@
 from nemo.collections.nlp.models.language_modeling.megatron_gpt_model import MegatronGPTModel
 from nemo.collections.nlp.models.nlp_model import NLPModel
 from nemo.collections.nlp.modules.common.megatron.megatron_init import initialize_model_parallel_for_nemo
-from nemo.collections.nlp.modules.common.prompt_encoder import PromptEncoder
+from nemo.collections.nlp.modules.common.t5_prompt_encoder import PromptEncoder
```

Comment:
Is this model still working? maybe we should delete this one if it is broken. 
In the future, let's add the feature that the model can only choose from specified set of labels for classification tasks in the prompt learning model.